Mathlib/Topology/Basic.lean

```diff
@@ -1359,6 +1359,10 @@ theorem IsOpen.preimage (hf : Continuous f) {t : Set Y} (h : IsOpen t) :
     IsOpen (f ⁻¹' t) :=
   hf.isOpen_preimage t h
 
+lemma Equiv.continuous_symm_of_isOpenMap (e : X ≃ Y) (h : IsOpenMap e) : Continuous e.symm := by
+  simp_rw [continuous_def, ← Set.image_equiv_eq_preimage_symm]
+  exact h
```

Comment:
How about making this an `Iff`?
```suggestion
lemma Equiv.continuous_symm_iff (e : X ≃ Y) : Continuous e.symm ↔ IsOpenMap e := by
  simp_rw [continuous_def, ← Set.image_equiv_eq_preimage_symm, IsOpenMap]

lemma Equiv.isOpenMap_symm_iff (e : X ≃ Y) : IsOpenMap e.symm ↔ Continuous e := by
  simp_rw [← Equiv.continuous_symm_iff, Equiv.symm_symm]
```

Reply:
You should probably use these lemmas to golf

https://github.com/leanprover-community/mathlib4/blob/5bf0f3993d81f682aab73c6129b25b49da7144fa/Mathlib/Topology/Homeomorph.lean#L426-L435

too